lib/awsapi/config.go

```diff
@@ -211,14 +211,22 @@ func (cfg *Config) getAPICredentials() (*credentials, error) {
 		}
 		return cfg.getRoleWebIdentityCredentials(string(token), cfg.irsaRoleARN)
 	}
+	token := os.Getenv("AWS_CONTAINER_AUTHORIZATION_TOKEN")
```

Comment:
According to the following doc https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html

 AWS sets the following env variables for pod Identity:
```
When configured, Amazon EKS sets the AWS_CONTAINER_CREDENTIALS_FULL_URI and AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE environment variables that the SDKs and tools use to obtain credentials.
```

 ECS authorization doesn't support that.

Reply:
It'd be great to add this link to the method comment and annotate relative and full path env variables with corresponding links to ECS taks IAM role and EKS pod identity.

Reply:
added link to AWS container credentials

Reply:
What about `AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE`? As far as I understand from AWS docs it doesn't set `AWS_CONTAINER_AUTHORIZATION_TOKEN` env variable. It's only fallback option for some comparability. 

 See the following blog post https://aws.amazon.com/blogs/containers/amazon-eks-pod-identity-a-new-way-for-applications-on-eks-to-obtain-iam-credentials/

Reply:
Logic should be the following:

if `AWS_CONTAINER_CREDENTIALS_FULL_URI` is set - read content of `AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE` and use it as token.

Reply:
I'm relying on SDK code
https://github.com/aws/aws-sdk-go-v2/blob/6d110cbf3144451a2aef547a785810eb3aa99bca/config/resolve_credentials.go#L184